### ISSUE TITLE
Can't build after perl version 5.26.

### ISSUE THREAD

**Issue by aero** [melezhik/outthentic#11]
```
Configuring M/ME/MELEZHIK/Outthentic-0.3.17.tar.gz with Makefile.PL
CPAN: CPAN::Reporter loaded ok (v1.2018)
Can't locate inc/Module/Install.pm in @INC (you may need to install the inc::Module::Install module) (@INC contains: .... 
```
Please see

http://blogs.perl.org/users/todd_rinaldo/2017/05/reviewing-perl-5-in-inc-at-the-perl-toolchain-summit.html
https://rt.cpan.org/Public/Bug/Display.html?id=120850

**Comment by spigell** [comment]
@melezhik I reported that in https://github.com/melezhik/sparrowdo/issues/6#issuecomment-312370357. 
There is a hack in sparrowdo for that - set PERL_USE_UNSAFE_INC=1 env variable. Also this env var sets in bootstrap.sh in https://sparrowhub.org/bootstrap.sh. 
Maybe there is another way to fix the problem on perl>5.26?

@aero Greetings! You can
1) install inc::Module::Install  manually and after that try to install outthentic again
2) set PERL_USE_UNSAFE_INC=1 env vars before installation of outthentic

3) Use bootstrap script for sparrow (https://sparrowhub.org/bootstrap.sh) 
Sparrow relies on outthentic so it will be installed. Info for sparrow - https://github.com/melezhik/sparrow

**Comment by melezhik** [comment]
Hi @aero . As workaround you can use method suggested by @spigell , I am leaving this issue open though as I'd like to provide more permanent solution for this. Let me know if it works for you,

Thanks

**Comment by melezhik** [comment]
fixed in 0.4.1 please check
